### ISSUE TITLE
embedding后加dropout

### ISSUE THREAD

**Issue by KaiyuanGao** [Determined22/zh-NER-TF#83]
Hi，
想请问在lookup_layer中对embedding后进行dropout操作的原因？
效果比不加dropout好在哪呢？
谢谢~
` 
self.word_embeddings = tf.nn.dropout(word_embeddings, self.dropout_pl)`

**Comment by 1311101YZD** [comment]
防止过拟合
